test/integration/https.spec.ts

```diff
@@ -419,4 +419,73 @@ describe("When configured for HTTPS", () => {
             });
         });
     });
-});
\ No newline at end of file
+
+    describe.only("with TLS version restrictions", () => {
+        const server = getLocal({
+                https: {
+                  keyPath: './test/fixtures/test-ca.key',
+                  certPath: './test/fixtures/test-ca.pem',
+                  tlsServerOptions: {
+                    minVersion: 'TLSv1.2'
+                  }
+                }
```

Comment:
Indentation has gone all funky here

Reply:
have another look? if not happy, would you let me know how you lint the project? i was expecting to find a lint script to run `npm run lint`.